engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetFboWriteMask.java

```diff
@@ -54,70 +49,55 @@
      * @param renderToColorBuffer A boolean indicating whether the Color buffer of the given FBO should be written to.
      * @param renderToDepthBuffer A boolean indicating whether the DepthStencil buffer of the given FBO should be written to.
      * @param renderToLightBuffer A boolean indicating whether the Light Accumulation buffer of the given FBO should be written to.
-     * @param fboName A ResourceUrn identifying the FBO whose render masks have to be modified - usually only the writeOnlyGBuffer FBO.
-     * @param fboManager The FBOManager responsible for managing the given FBO.
+     * @param fbo
      */
-    public SetFboWriteMask(boolean renderToColorBuffer, boolean renderToDepthBuffer, boolean renderToLightBuffer, ResourceUrn fboName, BaseFBOsManager fboManager) {
+    public SetFboWriteMask(boolean renderToColorBuffer, boolean renderToDepthBuffer, boolean renderToLightBuffer, FBO fbo) {
```

Comment:
Hmmm... I don't remember the rational behind that particular request. Admittedly it's not a big deal and we can leave it as it is. Somehow in this context it feels the fbo should go first: Set-Fbo-WriteMask provides the order somehow.